hcx-onboard/src/main/java/org/swasth/hcx/services/OnboardService.java

```diff
@@ -1171,4 +1171,16 @@ private void updateIdentityStatus(String email, String applicantCode, String ver
         postgreSQLClient.execute(query);
     }
 
+    @Async
+    public CompletableFuture<Map<String, Object>> asyncMockParticipant(HttpHeaders headers, String action, CompletableFuture<Map<String, Object>> mockDetails, Map<String, Object> participantDetails) {
```

Comment:
Why this method required? Implement async execution for creatMockParticipant method only.